modules/svg/image_loader_svg.cpp

```diff
@@ -35,6 +35,14 @@
 
 #include <thorvg.h>
 
+#ifdef __AVX__
+#ifdef _MSC_VER
+#include <intrin.h>
+#else
+#include <x86intrin.h>
+#endif
+#endif
```

Comment:
Replicating these lines to many files isn't a good idea. Should I isolate SIMD code into a different file?

Instead of writing intrinsics, using SIMD wrapper libraries such as https://github.com/p12tic/libsimdpp or https://github.com/simd-everywhere/simde might be good.